app/views/propertyrepresentation/appoint/confirmation.scala.html

```diff
@@ -36,7 +36,7 @@
     <p class="govuk-body">@messages("propertyRepresentation.confirmation.emailSent")</p>
 
     <h2 id="whatHappensNext" class="govuk-heading-m">@messages("propertyRepresentation.confirmation.whatHappensNext")</h2>
-    <p id="whatHappensNextP1" class="govuk-body">@messages("propertyRepresentation.confirmation.whatHappensNext.p1, agentName")</p>
+    <p id="whatHappensNextP1" class="govuk-body">@messages("propertyRepresentation.confirmation.whatHappensNext.p1", agentName)</p>
```

Comment:
This naming convention is not good (adding P1 and such like). Ids should tell you about the thing, not where it is currently or what html tag it currently uses. We do it all over the place, especially in the messages file, but it's not a good idea.

Reply:
🦅 👁 Dazza. That's a good catch.

Reply:
also, i don't think individual paragraphs (p tags) need an id. I thought only inputs or buttons or possibly divs might need it. only things the tests might need to be able to directly target.

Reply:
I agree with you @darrell-plant. It ended up with that name because I couldn't come up with a better name and I wanted to move on.

Redarding @peteharpersc's point. Im only adding the id's to aid testing so I will take a steer from Colin/Vijay on what elements they want to have ids.

Reply:
the confirmation page is different for each of the journeys in the new agent relationship stuff. So I would imagine that the QAs would want to confirm the text on the screen

Reply:
may be we can have a general chat with the QAs re what elements should get ids